src/modules/Dropdown/DropdownMenu.js

```diff
@@ -32,6 +35,7 @@ DropdownMenu.propTypes = {
 
   /** Classes to add to the className. */
   className: PropTypes.string,
-}
 
-export default DropdownMenu
+  /** Will mark the menu as opened when nested in another menu */
+  opened: PropTypes.bool,
```

Comment:
Once this is updated, let's update the actual Dropdown usage of the DropdownMenu to use the prop instead of passing the `className`.

https://github.com/TechnologyAdvice/stardust/blob/master/src/modules/Dropdown/Dropdown.js#L872